src/app/components/homepage/homepage.component.html

```diff
@@ -0,0 +1,20 @@
+<app-carousel></app-carousel>
+<br><br>
+
+<div class="row">
+    <div class="col-sm-4 hometile"><img src="/assets/bed-home.jpg" alt=""></div>
+    <div class="col-sm-4 hometile"><img src="/assets/dining-home.jpg" alt=""></div>
+    <div class="col-sm-4 hometile"><img src="/assets/kitchen-home.jpg" alt=""></div>
+</div>
+
+<div class="hometile1-text">
+        <h4 style="margin-left: 21px;">Bedroom</h4>
+</div>
+<div class="hometile2-text">
+        <h4 style="margin-left: 21px;">Dining</h4>
+</div>
+<div class="hometile3-text">
+        <h4 style="margin-left: 21px;">Kitchen</h4>
```

Comment:
* indentation is incorrect
* inline styling is a bad practice - remove margin-left style and move to css file